src/map/js/helpers/LayersHelper.js

```diff
@@ -408,6 +408,38 @@ let LayersHelper = {
     return [features[0]];
   },
 
+  setTweetTemplates: function(features, mapPoint) {
+    let template, htmlContent, tweetId;
+    console.log(mapPoint);
+    features.forEach(item => {
+      let url = item.feature.attributes.link;
+      let indexId = url.lastIndexOf('/') + 1;
+      tweetId = url.substring(indexId);
+
+      if (app.mobile() === true) {
+        htmlContent = '<div id="tweet" class="tweet-container mobile"><div title="close" class="infoTweetWindow-close close-icon"><svg viewBox="0 0 100 100"><use xlink:href="#shape-close" /></use></svg></div></div>';
```

Comment:
This close icon looks good, but doesn't work. There is no click event registered with this new 'infoTweetWindow-close' class. 

Check out the dojoQuery we run on the 'infoWindow-close' class elsewhere in the LayersHelper.js. Here is where we add that onClick event that actually closes the infowindow. Doing something similar with your new class could work, but I would simply use the same old class we have always used, with additional classes on that for styling purposes.